### PR TITLE
Add IIS gce overview

### PR DESCRIPTION
**Pull request by Mrod1598**
This depends on the ops-agent IIS metric receiver https://github.com/GoogleCloudPlatform/ops-agent/pull/534

### CODE REVIEW

**Comment by jefferbrecht** [comment]
Can you crop the non-dashboard stuff out of dashboards/iis/iis-gce-overview-03.png? (bottom-right corner)

File: dashboards/iis/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+category: IIS
```
Comment (EvanSimpson) [inline dashboards/iis/metadata.yaml:1]: I'll note that the format for metadata.yaml is still being finalized so there's expectation for it to be included as of now. Feel free to leave or revert, I'll make any necessary adjustments later and outline the expected format in the contribution docs when it's ready.

Comment (Mrod1598) [reply dashboards/iis/metadata.yaml]: gotcha, reverted for now then.

**Comment by Mrod1598** [comment]
> Can you crop the non-dashboard stuff out of dashboards/iis/iis-gce-overview-03.png? (bottom-right corner)

Cropped

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**